### PR TITLE
fix(WI-quick-add): Remove redundant scrollbar from description

### PR DESCRIPTION
**Pull request by jarifibrahim**
This PR fixes https://openshift.io/openshiftio/openshiftio/plan/detail/1550

### CODE REVIEW

File: src/app/components/work-item-quick-add/work-item-quick-add.component.less [hunk 1]
```diff
@@ -40,10 +40,12 @@
...
       padding-right: 70px;
       padding-left: 42px;
-      textarea[class='f8-quickadd-desc'] {
-        max-height: 100px;
-        height: 26px !important;
-        padding-top: 0;
+      textarea.f8-quickadd-desc {
+        bottom: -25px;
+        max-height: 200px;
+        max-width: 407px;
```
Comment (SMahil) [inline src/app/components/work-item-quick-add/work-item-quick-add.component.less]: max-width is not needed

File: src/app/components/work-item-quick-add/work-item-quick-add.component.less [hunk 2]
```diff
@@ -40,10 +40,12 @@
...
+        position: absolute;
```
Comment (SMahil) [inline src/app/components/work-item-quick-add/work-item-quick-add.component.less]: why position absolute?

File: src/app/components/work-item-quick-add/work-item-quick-add.component.less [hunk 3]
```diff
@@ -94,10 +96,11 @@
     margin-left: 0;
     background: @color-pf-white;
     .Border(2px, solid, @color-pf-black-400);
-    textarea[class="f8-quickadd-desc"] {
+    textarea.f8-quickadd-desc {
```
Comment (SMahil) [inline src/app/components/work-item-quick-add/work-item-quick-add.component.less]: why change of syntax?

File: src/app/components/work-item-quick-add/work-item-quick-add.component.less [hunk 4]
```diff
@@ -124,6 +127,6 @@
 }
 @-moz-document url-prefix() {
   .f8-quickadd__wiblk-story {
-    textarea[class="f8-quickadd-desc"] { max-height: 26px !important; }
+    textarea.f8-quickadd-desc { max-height: 26px !important; }
```
Comment (SMahil) [inline src/app/components/work-item-quick-add/work-item-quick-add.component.less]: why change of syntax

File: src/app/components/work-item-quick-add/work-item-quick-add.component.less (hunk 1 above)
Comment (jarifibrahim) [reply src/app/components/work-item-quick-add/work-item-quick-add.component.less]: screenshot from 2017 10 17 00 24 31

^^This is why max-width is necessary. The description field is a `col-md-6` and title field is also a `col-md-6` Without max-width, the description field overlaps with the buttons.

File: src/app/components/work-item-quick-add/work-item-quick-add.component.less (hunk 2 above)
Comment (jarifibrahim) [reply src/app/components/work-item-quick-add/work-item-quick-add.component.less]: We want the description field to grow in vertically upward direction. That's why it's position has to absolute. Without this, the textarea will grow vertically downward.

File: src/app/components/work-item-quick-add/work-item-quick-add.component.less (hunk 3 above)
Comment (jarifibrahim) [reply src/app/components/work-item-quick-add/work-item-quick-add.component.less]: `textarea[class='f8-quickadd-desc']` isn't the correct way of selecting a textarea with class f8-quickadd-desc. If you use `textarea[class='f8-quickadd-desc']` none of the styles are reflected.

**Comment by SMahil** [comment]
Textarea is overflown from quick add container

screen shot 2017-10-17 at 12 55 47 pm

**Comment by jarifibrahim** [comment]
@SMahil This is the expected result, right? We want the textarea to grow vertically upward. Textarea has to overflow from quick-add-container, otherwise it won't grow :laughing:

**Comment by SMahil** [comment]
I don't think this is expected behaviour.It seems broken now. We can approach to UX team for better solution

**Comment by jarifibrahim** [comment]
Closing this PR since it does not fix the actual issue and rather breaks the UI.

**Pull request closed without merging**